src/draw.c

```diff
@@ -75,7 +75,39 @@ draw_chars(struct view *view, enum line_type type, const char *string, int lengt
 
 	set_view_attr(view, type);
 	if (len > 0)
-		waddnstr(view->win, string, len);
+	{
+		if (view_has_flags(view, VIEW_DIFF_LIKE) &&
+		    !opt_diff_show_signs &&
+		    view->col == 0 &&
+		    (type == LINE_DIFF_ADD || type == LINE_DIFF_DEL || type == LINE_DEFAULT) &&
```

Comment:
LINE_DEFAULT is also used for the commit description so it is impacted as well. 

You also miss the case of merge commits where diff --cc is used. In this case there are two columns of +- (try `tig show f7b4319` in tig repository). Unfortunately draw_chars is too low level to known about it. 

A while ago I gave it a try but didn't like it so it stayed in the box. If you want to have a look, I pushed it at https://github.com/koutcher/tig/tree/gh-855-hide-diff-signs.

Reply:
Thanks for the pointers!

In the repos I use at work there are no merge commits so I have completely missed that case :(

I will have a look at your branch and make a decision if this patch can be further developed or not.